files/image_config/monit/restart_service

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
Suggest changing to `#!/usr/bin/env python3` until we fully remove Python 2. This will ensure the script is Python 3-compliant and we will not encounter unexpected issues when we remove Python 2.

Reply:
Updated!